src/root.tsx

```diff
@@ -12,7 +12,8 @@ import {
     Scripts,
     Title
 } from "solid-start";
-import "./root.css";
+// eslint-disable-next-line
+import "@mutinywallet/ui/style.css";
```

Comment:
eslint hates this import, any ideas on how we can appease it?

Reply:
sry :(